### PR TITLE
Feature/support 5.6

### PR DESCRIPTION
**Pull request by joshjacks**
Updated to the package to work with Laravel 5.6. This will break compatibility with 5.5.

### CODE REVIEW

**Comment by luceos** [comment]
Tests are failing because L 5.6 needs 7.1+; this has to be modified inside the `.circleci/config.yml` file.

Question here is how and if we're going to support future versions. On discord Jiri mentioned supporting latest LTS and latest version of Laravel. Which makes a lot of sense.

File: assets/configs/webserver.php [hunk 1]
```diff
@@ -105,7 +105,7 @@
         /**
          * The php sock to be used.
          */
-        'php-sock' => 'unix:/var/run/php/php7.0-fpm.sock',
```
Comment (luceos) [inline assets/configs/webserver.php:108]: good catch 👍

**Comment by luceos** [comment]
Relates and closes #383 , #384

**Comment by Thijmen** [comment]
Is it also possible to run the tests on L5.5? I have no experience with maintaining several laravel versions..

**Comment by luceos** [comment]
There are several ways to maintain multiple versions.

- Do not support older versions, this is something @spatie is very good at 😁 I don't want to do this because creating an implementation (saas) can take quite a long time, it would feel horrible to have finished your app only to find this package is no longer patching issues.
- Maintain and work on both versions inside the same branch. This requires conditionals and other "hacks" which I seriously have a distaste for.
- Maintain and work on both versions using different versions, this is what I did before branches 3.x, 4.x and 5.x for laravel 5.3, 5.4 and 5.5. Getting new stuff in is a nightmare in this scenario.
- Work on the latest version and patch the older version. When a serious issue is found we can check out a tag, patch and tag. All new functionality and overhauls are done on the master (or primary branch). This will by default make all existing implementation out of date, I'm not too happy about this either.

Multi version support in all scenarios require a lot more work and thus time. With the current patron donations I still cannot dedicate enough time to work on tenancy at a steady pace. I wish this was different though.

 My current approach has been to support only the latest LTS version (hence dropping branches 3.x and 4.x from the scope completely, I even deleted all tagged versions).

**Comment by dahabit** [comment]
waiting that pull request to use is with laravel 5.6 new project

**Comment by luceos** [comment]
@Thijmen is going to take care of adding 5.6 support with backward compatibility. This will be part of the upcoming 5.1.0 release.

**Comment by luceos** [comment]
@Maveriks, Josh thank you for doing the necessary work to get this in. Good job, much appreciated!

**Pull request merged**